css/_navigate_section_list.scss

```diff
@@ -4,7 +4,7 @@
     line-height: 20px;
     color: $welcomePageTitleColor;
     text-align: left;
-    font-family: 'open_sanslight', Helvetica, sans-serif;
+    font-family: $baseFontFamily;
```

Comment:
Do you need to declare a font-family at all? Does it not automatically inherit?

Reply:
Good question! I saw that it doesn't use the variable and change it... I'll try to remove it and check.

Reply:
It turned out we are not using the navigate section list right now. I left it some time ago because I was thinking it can be useful in future...

Reply:
Let's leave it like that. I guess soon we can even remove it...

WDYT?

Reply:
You should probably just remove it. You can look in git history if you think it'll be useful.